README.md

```diff
@@ -1,8 +1,12 @@
 # Lab
 
 The Stellar Lab is a suite of tools to help one learn about exploring the
-Stellar network. See it in action:
-[https://lab.stellar.org/](https://lab.stellar.org/).
+Stellar network.
+
+The `main` branch is deployed to
```

Comment:
isn't it master @quietbits

Reply:
I'm planning to switch to `main` later today.